cluster/oke.go

```diff
@@ -79,7 +85,17 @@ func (o *OKECluster) CreateCluster() error {
 		return err
 	}
 
-	return cm.ManageOKECluster(&o.modelCluster.OKE)
+	err = cm.ManageOKECluster(&o.modelCluster.OKE)
+	if err != nil {
+		return errors.Wrap(err, "error creating cluster")
+	}
+
+	err = o.setClusterAdminRights("cluster-creator-admin-right")
+	if err != nil {
+		return errors.Wrap(err, "error get/create clusterrolebinding")
```

Comment:
Errors in the `setClusterAdminRights` already Wrap original errors. `errors.Wrap` does two things: attaches stack trace and an additional message to the error. By calling Wrap again, the original stack trace gets lost.

For subsequent error wrapping use `errors.WithMessage` instead which attaches an additional message only.